### PR TITLE
Change Mo topic check

### PR DESCRIPTION
**Pull request by AbhayGoyal**
Changed the topic and timezone for Mo Akhshik

### CODE REVIEW

File: _data/speakers.yml
Comment (canyon289) [inline _data/speakers.yml]: Per oriols suggestion I would change this naming to what was said in slack

File: _data/speakers.yml
Comment (canyon289) [inline _data/speakers.yml]: Can we add the summary as well?

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Yup. I guess we can change it in the sheet as well?

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Did change the summary. pushed it

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: $\sim$3 pat in the summary looks a bit unnecessary. Should I remove it?

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]: can you remove the latex things? Unluckily they won't work on the website and will look weird:

* `$\sim$3` -> `~`
* `\texttt{pymc3}` -> PyMC3

**Comment by canyon289** [comment]
@AbhayGoyal I have to jump off again but basically here's what will happen

Once this is good Oriol will merge this pull request into the main code branch. I dont actually think its going to show up on the website yet because we havent coded it, but youve basically gotten it ready so once were ready to make the changes on the website all the text will already be available!

As youve got time itd be great if you can add however many more. What I would suggest for the next few is making a pull request early so people know youre working, and in these comments mention which speakers youll be working on so we can coordinate without running into each other

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Changed.

File: _data/speakers.yml
Comment (canyon289) [reply _data/speakers.yml]: Its fine in the sheet

**Comment by canyon289** [comment]
This is great! I'll let @OriolAbril review since hes working on the grouping but I have no specific suggestion. Thrilled youre making so much progress on this pull request :)

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]:
```suggestion
  image: Viola.jpg
```

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]:
```suggestion
  proposal_summary: Understanding how the most massive galaxies rapidly formed and quenched when Universe was only ~3 billion years old is one of the major challenges of extragalactic astronomy. In this talk, I will discuss how to improve our understanding of massive galaxy formation by combining the spectro-photometric observations of the Hubble and Spitzer Space Telescopes for strong gravitationally lensed galaxies. In particular, a multi-level regression model is built that can fit all multi-wavelength data for a range of instruments within a hierarchical Bayesian framework to constrain the properties of the stellar populations. The details of how this model is implemented using PyMC3, as well as the estimates of the posteriors of all parameters of interest and nuisance parameters will be highlighted.
```

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]:
```suggestion
  time_zone: Africa/Asia/Europe
```

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]: Everybody must have a keyword, if he takes part in 2 talks he should have the keyword of the "main" talk.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]: The keyword needs to be one per talk, otherwise the grouping won't work. Here however, Laura has `Laukes` and below Michael Osthege has `Micege`. They are giving a talk between the two. In order to be able to group them, they should both have the same keyword, maybe `calibr8` or anything else, but it must be one per talk, not one per speaker. Sorry to be so nitpicky, but otherwise the grouping won't work

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml:226]: mlda multilevel sampler has 3 speakers, all should have the same keyword.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]: mlda multilevel sampler has 3 speakers, they should all have the same keyword.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml:374]: Sequential monte carlo talk has 2 speakers, both should have the same keyword.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml]: mlda multilevel sampler has 3 speakers, all should have the same keyword.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml:459]: Michael should have the same keyword as Laura, the first and longer comment above.

File: _data/speakers.yml
Comment (OriolAbril) [inline _data/speakers.yml:506]: Sequential monte carlo talk has 2 speakers, both should have the same keyword.

**Review by OriolAbril — COMMENTED** [review]
Thanks for all the work! This looks good.

I have added a couple suggestions to fix typos and very minor changes and then some more structural changes in order to get the grouping to work.

Grouping needs the following:
* All speakers need to have `timezone`, `track` and `keyword` filled, keynote speakers, people giving 2 talks, everyone
* the `keyword` field must be one per talk, otherwise we can't group speakers of the same talk together. I have commented on some of them to give some more clear and specific example, but there may be more talks given by multiple people.

**Comment by canyon289** [comment]
> Thanks for all the work! This looks good.
> 
> I have added a couple suggestions to fix typos and very minor changes and then some more structural changes in order to get the grouping to work.
> 
> Grouping needs the following:
> 
> * All speakers need to have `timezone`, `track` and `keyword` filled, keynote speakers, people giving 2 talks, everyone
> * the `keyword` field must be one per talk, otherwise we can't group speakers of the same talk together. I have commented on some of them to give some more clear and specific example, but there may be more talks given by multiple people.

@OriolAbril Not all speakers will have a timezone. Elizaveta for example didnt commit to a synchronous Q&A, which is totally acceptable, so she wont have one. 

There are also a couple missing of timezones for people that havent responded yet so there will be some blanks for now

**Comment by OriolAbril** [comment]
We should then add a placeholder or give a name to not taking part in the live Q&A so they are all grouped together at the end or something like this

**Comment by canyon289** [comment]
Ok lets say `To be determined` for now, and then we fill in as they come

**Comment by hectormz** [comment]
I think `keyword` got copied as `keywork` throughout the file

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Done this, added vincent's keyword to Matthijs

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Did this

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Done this

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml:374]: Done

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml]: Done

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml:459]: Changed the keywork

**Comment by AbhayGoyal** [comment]
I am still confused about Matthijs Brouns. He is giving 2 talks, one himself and the other is with Vincent, what would his keywork be then?

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml:506]: Changed the keywork

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:79]:
```suggestion
  keyword: mohik
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:102]:
```suggestion
  keyword: Alerra
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:121]:
```suggestion
  keyword: Aguelo
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:142]:
```suggestion
  keyword: Vindam
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:163]:
```suggestion
  keyword: Hesehr
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:205]:
```suggestion
  keyword: Camlon
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:226]:
```suggestion
  keyword: Timwell
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:248]:
```suggestion
  keyword: Allney
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:270]:
```suggestion
  keyword: Micege
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:291]:
```suggestion
  keyword: Michns
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:312]:
```suggestion
  keyword: Saymar
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:332]:
```suggestion
  keyword: Junlao
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:352]:
```suggestion
  keyword: Timwell
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:374]:
```suggestion
  keyword: Osvtin
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:395]:
```suggestion
  keyword: Hesehr
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:416]:
```suggestion
  keyword: Timwell
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:437]:
```suggestion
  keyword: Quayen
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:459]:
```suggestion
  keyword: Micege
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:482]:
```suggestion
  keyword: Lucpaz
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:506]:
```suggestion
  keyword: Pedrez
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:526]:
```suggestion
  keyword: Alidri
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:550]:
```suggestion
  keyword: Evdaka
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:571]:
```suggestion
  keyword: Vindam
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:591]:
```suggestion
  keyword: Zheang  
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:610]:
```suggestion
  keyword: Thocki
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:631]:
```suggestion
  keyword: Robkov
```

File: _data/speakers.yml
Comment (hectormz) [inline _data/speakers.yml:653]:
```suggestion
  keyword: Eliova
```

**Review by hectormz — CHANGES_REQUESTED** [review]

File: _data/speakers.yml
Comment (AbhayGoyal) [reply _data/speakers.yml:79]: Hey Hector, I had done that on purpose actually. Should I change them all?

**Comment by OriolAbril** [comment]
I think the `keywork` is probably a typo on my side, I will merge this and change locally from the text editor which will be faster, thanks @AbhayGoyal and @hectormz. I'll try to rebase and solve the merge issues on your other PR too

**Comment by hectormz** [comment]
Got it, sounds good @OriolAbril !

**Pull request merged**
